### ISSUE TITLE
Frontend Routes & Components

### ISSUE THREAD

**Issue by pvulakh** [jessicaf1/pinthefail#5]
Good work on this! A few changes to make:

- [ ] `users/:id/boards` and `users/:id/pins` both should render the ProfileComponent as well as their respective main components
- [ ] `/users/:id` and `/users/:id/boards` look the same, so have the first route also render the UserBoardsIndex which has UserBoardIndexItems
- [ ] `/users/:userId/boards/:boardId` should also render a PinIndex with PinIndexItems
- what is `boards`? I can't find the corresponding route on pinterest
- [ ] remove `/boards/:boardId/edit` route since editing a board shows up as a modal (not as a separate front end route)
- don't worry about `/boards/pins` or `/pins`, your feed will function as that 

Let me know if you have any questions!

**Comment by jessicaf1** [comment]
-for "/users/:userId/boards/:boardId should also render a PinIndex with PinIndexItems" - that means it should render the pinIndex/pinIndexitems for everything on that specific board? 
-In general, I'm a bit confused about setting up these routes - for the above example, does that mean that there is the "/users/:userId/boards/:boardId" Route set up on the App page, and then the corresponding presentational component renders pin index and pin index items?

**Comment by pvulakh** [comment]
Yes, for `/users/:userId/boards/:boardId` the PinIndex will render the pins for that specific board. That does indeed mean there is a route corresponding to that path that renders the PinIndex with the proper pins!
